[Notifications] Update configure-webhooks.md (#10832)
cloudflare/cloudflare-docs: content/notifications/create-notifications/configure-webhooks.md

      <td>User decides.</td>
   </tr>
  </tbody>
</table>

> {{<Aside type="note">}}
> Note regarding the configuration of Slack webhook using API or Terraform
> You need to explicitly use the `secret` parameter to define the secret, instead of leaving it at the end of the webhook URL.
> Example for Terraform:
> resource "cloudflare_notification_policy_webhooks" "example" {
>   account_id = "<account_id>"
>   name       = "Slack Webhook"
>   url        = "https://hooks.slack.com/services/T00000000/B00000000"
>   secret     = "<secret>"
> }
> {{</Aside>}}

Reviewer: ```suggestion
The configuration of a Slack webhook using API or Terraform requires you to explicitly use the `secret` parameter to define the secret, instead of leaving it at the end of the webhook URL.

```tf
---
header: Terraform example
---
resource "cloudflare_notification_policy_webhooks" "example" {
  account_id = "<account_id>"
  name       = "Slack Webhook"
  url        = "https://hooks.slack.com/services/T00000000/B00000000"
  secret     = "<secret>"
}
```

```
might be better to not have this as a note
